tests/kani/Enum/neg_discriminant.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// This test checks that enums with negative discriminants are handled correctly
+
+enum Foo {
+    A = -500,
+    B = -200,
+    C = -100,
+    D = 0,
+    E = 1,
+    F = 256,
+}
+
+#[kani::proof]
+fn main() {
```

Comment:
nit: I think we should stop using main() and add a more descriptive name to the harness now that we run them by default.